pkg/operator/ceph/csi/spec.go

```diff
@@ -756,6 +758,46 @@ func (r *ReconcileCSI) stopDrivers(ver *version.Info) error {
 	return nil
 }
 
+func (r *ReconcileCSI) deleteCSIOperatorResources(clusterNamespace string) {
+	csiCephConnection := &csiopv1a1.CephConnection{}
+
+	err := r.client.DeleteAllOf(r.opManagerContext, csiCephConnection, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: clusterNamespace}})
```

Comment:
There might be clusters across multiple namespaces. Will this be called separately for each cluster namespace?